includes/definitions/microsemipdsine.yaml

```diff
@@ -0,0 +1,7 @@
+os: microsemipdsine
+text: 'Microsemi PowerDsine Midspan PoE'
+type: network
+icon: microsemi
+discovery:
+    - sysObjectId:
+        - .1.3.6.1.4.1.7428.1.1.1.23
```

Comment:
I will use `.1.3.6.1.4.1.7428.1.1.1.`
After searching around, it looks like PowerDsine PoE's go from `..1.1.1.1` to `...1.1.1.23`